core/templates/components/entity-creation-services/skill-creation.service.ts

```diff
@@ -70,19 +70,24 @@ angular.module('oppia').factory('SkillCreationService', [
         }
         skillCreationInProgress = true;
         AlertsService.clearWarnings();
+        // $window.open has to be initialized separately since if the 'open new
+        // tab' action does not directly result from a user input (which is not
+        // the case, if we wait for result from the backend before opening a new
+        // tab), some browsers block it as a popup. Here, the new tab is created
+        // as soon as the user clicks the 'Create' button and filled with URL
+        // once the details are fetched from the backend.
+        var newTab = $window.open();
```

Comment:
How will this look form the user's PoW? Will the empty tab be created first and then filled with the URL or will the tab be only shown once the URL is set?

Reply:
An empty tab is created first and then the URLis filled, so yeah there is a delay from thee user's PoV. I referred [this](https://stackoverflow.com/a/50249439/10764956) SO answer.

Reply:
Any specific reason for opening this in a new tab? I don't see this workflow for exploration creation.

Reply:
Actually, @seanlip suggested it when he was creating the new lessons on the server as it was more convenient.